engine/dlib/src/dlib/sys.cpp

```diff
@@ -898,10 +891,8 @@ namespace dmSys
             }
             *resource_size = asset_size;
             return RESULT_OK;
-        } else {
-            return RESULT_NOENT;
         }
-#else
+#endif
```

Comment:
I think these functions FileExists, ResourceSize and LoadResource should be able to find files on disc on all platforms, including Android.

This issue came after refactoring the archive loading code to use these functions as opposed to specific "fopen" et al.